e2e/testconfig/testconfig.go

```diff
@@ -360,6 +370,8 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string, cometCf
 	tmTomlOverrides["log_level"] = cometCfg.LogLevel // change to debug in ~/.ibc-go-e2e-config.json to increase cometbft logging.
 	configFileOverrides["config/config.toml"] = tmTomlOverrides
 
+	useNewGenesisCommand := cc.Binary == icadBinary && icadNewGenesisCommandsFeatureReleases.IsSupported(cc.Tag)
```

Comment:
It would be nice if it was possible to check if the command exists by using the specified binary. But it seems this is likely interchain test could have handled?

Reply:
I think this should be possible using `<bin> help --json` and parsing it as appropriate. I agree that this should be happening in interchaintest.